src/SFA.DAS.RequestApprenticeTraining.Infrastructure/Configuration/ApplicationConfiguration.cs

```diff
@@ -4,5 +4,7 @@ public class ApplicationConfiguration
     {
         public EmployerRequestApprenticeTrainingOuterApiConfiguration EmployerRequestApprenticeTrainingOuterApiConfiguration { get; set; }
         public FunctionsOptions FunctionOptions { get; set; }
+        public string EmployerRequestApprenticeshipTrainingBaseUrl { get; set; }
```

Comment:
How are these config's used?

Reply:
Code was missing.  Have added it to SendEmployerRequestsResponseNotification where they used when creating the request to the outer api